demo-shell/src/app/components/trashcan/trashcan.component.html

```diff
@@ -76,7 +76,9 @@
                     key="archivedAt"
                     title="DOCUMENT_LIST.COLUMNS.DELETED_ON">
                     <ng-template let-value="value">
-                        <span title="{{ value | date:'medium' }}">{{ value | adfTimeAgo: currentLocale }}</span>
+                        <div class="adf-trashcan-center-delete-column-ie">
```

Comment:
this is a fix in "demo shell" that other applications don't know about.

Reply:
the name of the class does not reflect what it actually does and is confusing given its applied to a date column.